src/main/java/dal/fatih/todoly/TaskRepository.java

```diff
@@ -0,0 +1,164 @@
+package dal.fatih.todoly;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskRepository {
+
+    private ResultSet resultSet;
+    final DBConnection dbConnection = new DBConnection();
+    protected Connection connection = dbConnection.getConnection();
+    final PreparedStatement createPrepareStatement;
+    final PreparedStatement listPrepareStatement;
+    final PreparedStatement getPreparedStatement;
+    final PreparedStatement deleteTaskByIdPreparedStatement;
+    final PreparedStatement filterPreparedStatement;
+    final PreparedStatement getByTitleOrDesPreparedStatement;
+    private List<Task> tasks;
+
+    public TaskRepository() throws SQLException {
+
+        createTable();
+        createPrepareStatement =
+                connection.prepareStatement("insert into tasks " + "values (?,?,?,?,?)");
+        listPrepareStatement =
+                connection.prepareStatement("select id,taskid,title,description,duedate from tasks");
+        getPreparedStatement =
+                connection.prepareStatement("select taskid,title,description,duedate from tasks where taskid=? limit 1");
+        deleteTaskByIdPreparedStatement =
+                connection.prepareStatement("delete from tasks where taskid=?");
+        filterPreparedStatement =
+                connection.prepareStatement("select * from tasks where duedate between now() and ?");
+        getByTitleOrDesPreparedStatement =
+                connection.prepareStatement("select taskid,title,description,duedate from tasks where title like ? "
+                        + "or description like ?");
+    }
+
+    public void clearTable() {
+        try {
+            PreparedStatement preparedStatement = connection.prepareStatement("TRUNCATE TABLE tasks");
+            preparedStatement.execute();
+            connection.close();
+            preparedStatement.close();
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }

```

Comment:
This is only used for testing I guess. Code that is used for testing should be in the testing not here. Move this function to the test where ever you need to call it.